scripts/infra

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${OAR_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
```

Comment:
It seems as though this variable is only used once. Could be removed, if desired.